src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -1689,5 +1689,111 @@
         "Program.cs"
       ]
     }
+  },
+  "blazorwasm-empty": {
+    "None": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "_Imports.razor",
+        "App.razor",
+        "MainLayout.razor",
+        "Program.cs"
+      ]
+    },
+    "Hosted": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --hosted",
+      "Files": [
+        "Client/_Imports.razor",
+        "Client/App.razor",
+        "Client/MainLayout.razor",
+        "Client/Program.cs",
+        "Client/Pages/Index.razor",
+        "Client/Properties/launchSettings.json",
+        "Client/wwwroot/index.html",
+        "Client/wwwroot/css/app.css",
+        "Server/Properties/launchSettings.json",
+        "Server/appsettings.Development.json",
+        "Server/appsettings.json",
+        "Server/Program.cs"
+      ]
```

Comment:
Was the `Shared` content removed entirely?

Reply:
Yes, for the empty template I removed all contents of the `Shared` folder (which was just WeatherForecast.cs).

Reply:
In that case should we also remove the Shared `csproj` as well as other references to it (ie. `template.json`, `.sln`, etc.)? Or did you have something else in mind for it?

Reply:
I like the idea of keeping the Shared `csproj` and folder for templating purposes since this folder is also included in the default WebAssembly template. I can add this and update the tests as needed.